app/javascript/views/addUser/AddUser.test.js

```diff
@@ -34,6 +34,14 @@ describe('VerifyUser', () => {
     })
   })
 
+  describe('meta tag to disable hyperlinks for the phone number', () => {
+    it('verifies the meta tag exists', () => {
+      expect(wrapper.find('meta').length).toBe(1)
+      expect(wrapper.find('meta').props().name).toBe('format-detection')
+      expect(wrapper.find('meta').props().content).toBe('telephone=no')
```

Comment:
Ideally we just want one test (one expect) per 'it' block.  I think we can cover all of what we want to test here with a more specific find expression...
```suggestion
      expect(wrapper.find("meta[name='format-detection']").props().content).toBe('telephone=no')
```

Reply:
Updated!